<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>listbreak</class>
 <widget class="QDialog" name="listbreak">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>List Breakpoints</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>531</width>
     <height>323</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayoutBPList">
    <property name="sizeConstraint">
     <enum>QLayout::SetNoConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <item>
     <widget class="QLabel" name="ABP_Label">
      <property name="text">
       <string>Architecture Breakpoints</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="ABP_List"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutRM_ABP">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="ABP_removeButton">
        <property name="text">
         <string>Remove Selected Breakpoint</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="PBP_Label">
      <property name="text">
       <string>Program Breakpoints</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="PBP_List"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayoutRM_PBP">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="PBP_removeButton">
        <property name="text">
         <string>Remove Selected Breakpoint</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
